#Configuration file for laser shade occlusion manager
{
    occlusion_manager_number_lasers: 1,
    use_occlusion_geodescis_manager: true,
    occlusion_manager_visualization_enabled: false,
    occlusion_manager_max_range: 20,
    occlusion_self_occlusion_distance: 0.25,
    occlusion_neigbor_polygon_width: 0.2,
    max_misses_before_deletion: 5,
    max_misses_before_deletion_of_mature_track: 10,
    track_is_mature_after_total_num_matches: 97,
    occlusion_manager_subscriber_queue_size: 100,
    occlusion_manager_sync_buffer_size: 100,
    occlusion_allowed_duration_for_reappearance: 2.6,
    occlusion_manager_maximum_sync_slop: 0.1,
    occlusion_manager_minimum_absolute_velocity: 0.01,
    occlusion_geodesics_motion_variance: 1.3,
    occlusion_geodesics_plausibility_cut_off: 0.2,
    occlusion_geodesics_inertia_variance: 0.2,
    occlusion_geodesics_infimum_radius: 3,
    occlusion_geodesics_grid_cell_resolution_meter: 0.1,
    occlusion_geodesics_grid_dimension_forward_meter: 5.0,
    occlusion_geodesics_grid_dimension_backward_meter: 0.5,
    occlusion_geodesics_grid_dimension_width_meter: 3.0,
    occlusion_geodesics_detector_reliability: 0.92,
    occlusion_geodesics_number_of_assignments_before_acceptance: 4,
    occlusion_geodesics_allowed_orientation_difference: 0.5,
}
